docs/error-messages/compiler-warnings/compiler-warning-levels-3-and-4-c4244.md

```diff
@@ -8,29 +8,28 @@ ms.assetid: f116bb09-c479-4b4e-a647-fe629a1383f6
 
 'conversion' conversion from 'type1' to 'type2', possible loss of data
 
-An integer type is converted to a smaller integer type. This is a level-4 warning if *type1* is **`int`** and *type2* is smaller than **`int`**. Otherwise, it is a level 3 (assigned a value of type [__int64](../../cpp/int8-int16-int32-int64.md) to a variable of type **`unsigned int`**). A possible loss of data may have occurred.
+An integer type is converted to a smaller integer type. This is a level-4 warning if *type1* is **`int`**/**`unsigned int`** and *type2* is smaller than **`int`**/**`unsigned int`**, such as **`short`** and **`unsigned short`**. Otherwise, it is a level 3 if a value of type [__int64](../../cpp/int8-int16-int32-int64.md)/**`unsigned __int64`** is assigned to a variable of type **`int`**/**`unsigned int`**. A possible loss of data may have occurred due to a narrowing conversion, which may lead to unexpected effects.
 
-If you get C4244, you should either change your program to use compatible types, or add some logic to your code, to ensure that the range of possible values will always be compatible with the types you are using.
+If you get C4244, you should either change your program to use compatible types, or add some logic to your code, to ensure that the range of possible values will always be compatible with the types you are using. If the conversion is intended, an explicit cast can be added to silence the warning.
 
 C4244 can also fire at level 2; see [Compiler Warning (level 2) C4244](../../error-messages/compiler-warnings/compiler-warning-level-2-c4244.md) for more information.
 
-The conversion may have a problem due to implicit conversions.
-
 The following sample generates C4244:
 
 ```cpp
 // C4244_level4.cpp
 // compile with: /W4
-int aa;
-unsigned short bb;
+void foo(unsigned short num) {}
```

Comment:
Hi, we can't use the name 'foo' anymore. Can you change to myfunc, or something?